site/en/codelabs/openthread-border-router-ipv6-multicast/index.lab.md

```diff
@@ -283,6 +283,8 @@ Now, we should be able to reach both nRF52840 End Device 1 in the Thread network
 **Ping ff05::abcd on OTBR via the Wi-Fi interface:**
 
 ```console
+# Below is the command for linux-based host, equivalent command for Mac OS is:
+# "ping6 -h 5 -I wlan0 ff05::abcd" and it's usually not "wlan0" for Mac.
```

Comment:
Instead of putting these here, I'd recommend having the other example after this one.  So like one large command block:

```console
$ ping -6 -b -t 5 -I wlan0 ff05::abcd
PING ...

# If using MacOS, use this command. The interface is typically not "wlan0" for Mac.
$ ping6 -h 5 -I wlan0 ff05::abcd
```

Or you can just have a separate paragraph somewhere that mentions it.

Reply:
I updated to what you suggested. I didn't append the output of the command on Mac OS because the prefixes and addresses don't match the linux ones and it will cause confusion if I do add it there.